### PR TITLE
docs: prettier home page for the documentation portal

### PR DESCRIPTION
**Pull request by noandrea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,24 +46,10 @@ Links to presentations, discussions and interviews are available in
...
-available at:
-
-```url
-https://resolver-driver.cosmos-cash.app.beta.starport.cloud/identifier/
-```
-
-A universal resolver frontend is also available for testing and verification purposes at:
-
-- https://uniresolver.cosmos-cash.app.beta.starport.cloud
+To join or interact using the testnet, refer to the [testnet coordinates page](https://docs.cosmos-cash.app.beta.starport.cloud/Networks/testnet/)
```
Comment (barriebyron) [inline README.md:52]:
```suggestion
To join or interact using the testnet, see [testnet coordinates](https://docs.cosmos-cash.app.beta.starport.cloud/Networks/testnet/)
```

Comment (barriebyron) [reply README.md:52]: hi @noandrea let's change the "refer to the" to "see" and just use the descriptive link text

can we use a relative link here?

File: README.md [hunk 2]
```diff
@@ -46,24 +46,10 @@ Links to presentations, discussions and interviews are available in
 To get started and contribute to the project, visit the [ Technical setup](./TECHNICAL-SETUP.md) page and the
```
Comment (barriebyron) [inline README.md:46]:
```suggestion
To get started and contribute to the project, see [Technical Setup](./TECHNICAL-SETUP.md) and the
```

Comment (barriebyron) [reply README.md:46]: it's useful to have the title of the page as the descriptive link test, and the more informal "see" (we don't need to say "page")

File: docs/ABOUT.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: About Documentation
```
Comment (barriebyron) [inline docs/ABOUT.md:2]: I remember when we spoke... I was looking for a quick description of Cosmos Cash and found instead descriptions of the docs, let's see how we can abbreviate this information or move to another sub level, let's make the cosmos cash docs about the project, not the docs

File: docs/How-To/README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # How-to guides
 
-The [How-To](./) folder contains how-to guides for different aspects of the Comsos project. Additional resources can be found in the [seed scripts](../../scripts/seeds) folder. 
+The [How-To](./) folder contains how-to guides for different aspects of the Cosmos project. Additional resources can be found in the [seed scripts](https://github.com/allinbits/cosmos-cash/tree/main/scripts/seeds) folder.
```
Comment (barriebyron) [inline docs/How-To/README.md:3]:
```suggestion
The [How-To](./) folder contains how-to guides for different aspects of the Cosmos project. Additional resources are in the [seed scripts](https://github.com/allinbits/cosmos-cash/tree/main/scripts/seeds) folder. 

```

Comment (barriebyron) [reply docs/How-To/README.md:3]: "Cosmos project" do we mean a blockchain that is built using the Cosmos SDK or do we mean generally "Cosmos ecosystem"?

File: docs/How-To/chain_001_faucet.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
+# Get Testnet Tokens (Faucet)
+
+The Cosmos Cash native token denom is `cash`.  Assuming that a keypair has been created with the follwing command:
```
Comment (barriebyron) [inline docs/How-To/chain_001_faucet.md:4]:
```suggestion
The Cosmos Cash native token denom is `cash`.  

A keypair is created with the following command:
```

File: docs/How-To/chain_001_faucet.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+```sh
+cosmos-cashd keys add alice
+```
+
+to obtain obtain `cash` tokens for the alice account from the testnet faucet use the following command:
```
Comment (barriebyron) [inline docs/How-To/chain_001_faucet.md:10]:
```suggestion
To obtain obtain `cash` tokens for the alice account from the testnet faucet, use the following command:
```

Comment (barriebyron) [reply docs/How-To/chain_001_faucet.md:10]: let's use sentence grammar, and add that missing comma; the colon before the command is helpful for readers to know a command is next

File: docs/Networks/testnet.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+# Testnet
+
+A Cosmos Cash testnet is available for testing purposes, this document describes the Cosmos Cash testnet coordinates and
+the satellite projects URLs.
```
Comment (barriebyron) [inline docs/Networks/testnet.md:4]:
```suggestion
A Cosmos Cash testnet is available for testing purposes.

Use these Cosmos Cash testnet coordinates and satellite project URLs:
```

File: docs/Networks/testnet.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+The faucet endpoint is used to get tokens to pay for transaction gas.
+
+|            |                                                      |
+| ---------- | ---------------------------------------------------- |
+| Faucet URL | `https://faucet.cosmos-cash.app.beta.starport.cloud` |
+
+
+## DID Resolver
+
+The following are the URLs for ([DID](../Reference/GLOSSARY.md#decentralized-identifier-did)) resolution in the Cosmos Cash network
```
Comment (barriebyron) [inline docs/Networks/testnet.md:26]:
```suggestion
URLs for ([DID](../Reference/GLOSSARY.md#decentralized-identifier-did)) resolution in the Cosmos Cash network.
```

File: docs/Networks/testnet.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+|                       |                                                                         |
+| --------------------- | ----------------------------------------------------------------------- |
+| DID resolver driver   | https://resolver-driver.cosmos-cash.app.beta.starport.cloud/identifier/ |
+| DID resolver frontend | https://uniresolver.cosmos-cash.app.beta.starport.cloud                 |
+
+## SSI Agent Router
+
+The following are the URLs to connect to a SSI Agent that uses also Cosmos Cash as VDR
```
Comment (barriebyron) [inline docs/Networks/testnet.md:35]:
```suggestion
URLs to connect to an SSI Agent that uses also Cosmos Cash as a verifiable data registry (VDR).
```

Comment (barriebyron) [reply docs/Networks/testnet.md:35]: tip: use "an" for words that sound like they start "s"

File: docs/README.md [hunk 10]
```diff
@@ -1,39 +1,38 @@
...
-    - [Contributing](#contributing)
-    - [Reference](#reference)
+![](https://miro.medium.com/max/1000/1*8Wx44uvyJxpZUVS0WojMNw.png)
 
-## Overview
+### Summary
 
-In Cosmos Cash we use the  *Grand Unified Theory of Documentation* (David Laing) as described
-by [Divio](https://documentation.divio.com/) as a basis for our documentation strategy.
+Cosmos Cash is a protocol designed to be regulatory compliant that offers the same guarantees as traditional banking
```
Comment (barriebyron) [inline docs/README.md:12]: hallelujah! thank you
```suggestion
Cosmos Cash is a protocol designed to be regulatory compliant that offers the same guarantees as traditional banking
```

File: docs/README.md [hunk 11]
```diff
@@ -1,39 +1,38 @@
...
+management by leveraging W3C specifications for decentralized identifiers and verifiable credentials.
 
-This approach outlines four specific use cases for documentation:
+### Research paper
 
-* [Tutorials](./Tutorials/README.md)
-* [How-Tos](./How-To/README.md)
-* [Explanation](./Explanation/README.md)
-* [Reference](./Reference/README.md)
+For more information on the research behind the Cosmos Cash protocol, please look at the Cosmos Cash research paper:
```
Comment (barriebyron) [inline docs/README.md:19]:
```suggestion
For more information on the research behind the Cosmos Cash protocol, see the Cosmos Cash research paper:
```

Comment (barriebyron) [reply docs/README.md:19]: oops let's not use please 
https://developers.google.com/style/tone#politeness-and-use-of-please

File: docs/Reference/GLOSSARY.md [hunk 12]
```diff
@@ -2,60 +2,92 @@
...
+transfer of virtual assets; iv. safekeeping and/or administration of virtual assets or instruments enabling control over
+virtual assets; and v. participation in and provision of financial services related to an issuer’s offer and/or sale of
+a virtual
+asset.” [FATF guidance to the G20](https://www.fatf-gafi.org/publications/virtualassets/documents/report-g20-so-called-stablecoins-june-2020.html)
 
+## Z
 
+### Zero-knowledge Proof (ZKP)
 
+In cryptography, a zero-knowledge proof or zero-knowledge protocol is a method by which one party (the prover) can prove to another party (the verifier) that a given statement is true, without conveying any information apart from the fact that the statement is indeed true. The essence of zero-knowledge proofs is that it is trivial to prove that one possesses knowledge of certain information by simply revealing it; the challenge is to prove such possession without revealing the information itself or any additional information ([wikipedia](https://en.wikipedia.org/wiki/Zero-knowledge_proof)).
```
Comment (barriebyron) [inline docs/Reference/GLOSSARY.md:91]:
```suggestion
In cryptography, a zero-knowledge proof or zero-knowledge protocol is a method by which one party (the prover) can prove to another party (the verifier) that a given statement is true, without conveying any information apart from the fact that the statement is indeed true. The essence of zero-knowledge proofs is that it is trivial to prove that one possesses knowledge of certain information by simply revealing it; the challenge is to prove such possession without revealing the information itself or any additional information ([Zero-knowledge proof on wikipedia](https://en.wikipedia.org/wiki/Zero-knowledge_proof)).
```

Comment (barriebyron) [reply docs/Reference/GLOSSARY.md:91]: descriptive link text must be the title of the page you land on, wdyt about this construction? (Wikipedia was too vague)

**Review by barriebyron — APPROVED** [review]
so much better for the home page, let's see if we can't get that docs about docs to a sub-level or somewhere other than the landing page

yay! good improvements

File: README.md (hunk 1 above)
Comment (noandrea) [reply README.md:52]: no, we have to have the fully qualified URL

**Pull request merged**